packages/pigeon/lib/dart_generator.dart

```diff
@@ -2,9 +2,13 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:code_builder/code_builder.dart' as cb;
+import 'package:collection/collection.dart' as collection;
```

Comment:
tl/dr; I removed this import and changed my usages of `mapIndexed` to `indexMap`.

But yes, I was using `mapIndexed` from `collection` instead of `indexMap` from the `functional.dart`. And these imports collide because they have a method called `mergeMaps`. I was originally using a different method from `collection`, but it doesn't look like I'm using it anymore. So i switched to just use the method from `functional.dart`. I do think we should eventually remove the methods from `functional.dart` that are already provided by the `collection` package.